yt/geometry/coordinates/cylindrical_coordinates.py

```diff
@@ -211,7 +211,7 @@ def sanitize_width(self, axis, width, depth):
             width = [2.0 * np.pi * self.ds.domain_width.uq,
                      self.ds.domain_width[z_ax]]
         elif name == "theta":
-            width = [self.ds.domain_right_edge[r_ax],
+            width = [self.ds.domain_width[r_ax],
```

Comment:
This is because for cylindrical geometry, Rmin is not always zero. When Rmin is larger than zero, the old code will give a larger width and create empty space.

Reply:
That's fair, yeah.  I am optimistic that with changes like those from @munkm that help with the different projections we can make this a bit easier to work with.